arch/x86/boot/compressed/sl_main.c

```diff
@@ -142,7 +148,7 @@ void sl_tpm_extend_pcr(struct tpm *tpm, u32 pcr, const u8 *data, u32 len)
 	early_sha1_update(&sctx, data, len);
 	early_sha1_final(&sctx, &sha1_hash[0]);
 	ret = tpm_extend_pcr(tpm, pcr, TPM_HASH_ALG_SHA1, &sha1_hash[0]);
-err:
+out:
```

Comment:
You don't need any label now, all the goto's are gone. It should be removed to prevent warnings.